plugin.video.fsgo/default.py

```diff
@@ -50,7 +52,7 @@ def addon_log(string):
         xbmc.log(msg=msg, level=xbmc.LOGDEBUG)
```

Comment:
@phil65 Thanks for the review. :) This is probably the wrong place for feature requests, but it would be nice if you could access this setting using xbmc.getCondVisibility.